cplusplus/VImage.cpp

```diff
@@ -64,7 +64,7 @@ to_vectorv( int n, ... )
 	va_list ap;
 
 	va_start( ap, n );
-	for( int i = 0; i < n; i++ )
+	for( size_t i = 0; i < n; i++ )
```

Comment:
`n` is an int, so I think I should be an int as well.

Reply:
I disagree. It's being used as an array index. The compiler may or may not promote it to a 64-bit integer on 64-bit systems.

Reply:
But `n` really is an int, if you look at the prototype. There's no point using `size_t` as the array index, the extra range can never be used.

C/C++'s rules around signed / unsigned comparisons are an almost endless source of bugs. In my opinion you should always use simply use int, unless there is a very good reason not to.